tools/driver-cxx/driver-cxx.cpp

```diff
@@ -306,26 +308,36 @@ int main(int argc, char **argv) {
 
   auto sandbox = GetProcessSandbox(SandboxOption);
 
+  std::vector<std::shared_ptr<mull::Filter>> filterStorage;
```

Comment:
I am under a strong impression that this storage can be encapsulated inside `mull::Filters`.

`mull::Filters` could then have a factory method for creating all filters from the CLI options and from needed dependencies such as JunkDetector.

Then the driver could be shorter and clear, with no details of filter creation.